test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
We can for now delete the unused file. If we need this file in future will create.